lib/style/module_directives.ex

```diff
@@ -65,18 +65,62 @@ defmodule Styler.Style.ModuleDirectives do
   """
   @behaviour Styler.Style
 
-  @directives ~w(alias import require)a
+  alias Styler.Zipper
+
+  @directives ~w(alias import require use)a
   @attr_directives ~w(moduledoc shortdoc behaviour)a
 
   # module names ending with these suffixes will not have a default moduledoc appended
   @dont_moduledoc ~w(Test Mixfile MixProject Controller Endpoint Repo Router Socket View HTML JSON)
   @moduledoc_false {:@, [], [{:moduledoc, [], [{:__block__, [], [false]}]}]}
 
-  def run({{:defmodule, def_meta, [name, [{mod_do, {:__block__, children_meta, children}}]]}, zipper_meta}, ctx) do
-    {directives, other} =
-      Enum.split_with(children, fn
+  def run({{:defmodule, _, children}, _} = zipper, ctx) do
+    [{_, _, aliases} = _module_name, [{mod_do, _module_body__move_focus_here!}]] = children
```

Comment:
are the `_named_var_comments_kinda` helpful? they just kind of annoy me, or at least the focus one does ^.^

Reply:
👍 merged